files/build_templates/docker_image_ctl.j2

```diff
@@ -10,6 +10,7 @@ start() {
         docker start -a {{docker_container_name}}
     else
         docker run {{docker_image_run_opt}} \
+            -v /var/run/redis:/var/run/redis:rw \
```

Comment:
I think it makes sense that all the containers can access the redis db. It aligns the idea that we store the state of the switch into the db. Although there might be some containers currently don't interact with the db, this makes it easy for us to add features and plugins to these containers.